heroes/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+
+from heroes.views import HeroViewSet
+
+
+router = DefaultRouter()
+router.register(r'heroes', HeroViewSet)
```

Comment:
It should be designed a little differently. Here you should have:
```suggestion
router.register(r'', HeroViewSet)
```

And the whole package should be mounted under `/heroes/`

Reply:
This router should be kept simple, it shouldn't know where it is mounted exactly. If you do this the way I recommend, you will be able to mount resource `/heroes/` under `''`, `'/somethingElse/`, `/blablab/`. You could think that this router can be unplugged and plugged everywhere you want to.